timesketch/models/sketch.py

```diff
@@ -351,6 +351,10 @@ class Analysis(GenericAttributeMixin, LabelMixin, StatusMixin, CommentMixin, Bas
     sketch_id = Column(Integer, ForeignKey("sketch.id"))
     timeline_id = Column(Integer, ForeignKey("timeline.id"))
     searchindex_id = Column(Integer, ForeignKey("searchindex.id"))
+    approach_id = Column(Integer, ForeignKey("investigativequestionapproach.id"))
```

Comment:
I know this is long and ugly, but to stay consistent in the file, please rename:
```suggestion
    investigativequestionapproach_id = Column(Integer, ForeignKey("investigativequestionapproach.id"))
```

Reply:
We are using `approach_id` already in the models: https://github.com/google/timesketch/blob/master/timesketch/models/sketch.py#L464 

Still want it to have changed? I'm fine with both.

Reply:
Let's keep the short version 👍🏼